### ISSUE TITLE
Worldquesttooltip faction

### ISSUE THREAD

**Issue by samyonair** [mikepauer/Carbonite#524]
Hi 
Great Work with Tooltip
Have a question if we can add Faction to tooltip because some factions have Quests everywere in Broken Isles and if you search them on map for the Daily Quest (4 from one faction) it would be helpful for that I use blizzmap at the moment.

greetings Samyonair

**Comment by IrcDirk** [comment]
YES we CAN ;) Future plans are to make list of world quest with filters and sorting.
